server/lib/analytics/tick-timer.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import analytics from './index';
+
+export function logTimingOfNextTick() {
+	/*
+	 * Call setImmediate once to enqueue a function at the end of the current
+	 * iteration of the event loop. From within the queued function, call
+	 * setImmedate a second time to schedule a callback to be called at the end
+	 * of the following event loop. As per the documentation, a setImmediate
+	 * callback scheduled within another setImmediate callback is guaranteed to
+	 * be run in the following iteration of the event loop.
+	 *
+	 * See:
+	 * https://nodejs.org/en/docs/guides/event-loop-timers-and-nexttick/
+	 * https://nodejs.org/api/timers.html#timers_setimmediate_callback_args
+	 */
+	setImmediate( () => {
+		const startTime = new Date();
+
+		setImmediate( () => {
+			analytics.statsd.recordTiming(
```

Comment:
This doesn't seem like it should work, given we're [stubbing out `analytics`.](https://github.com/Automattic/wp-calypso/blob/master/webpack.config.node.js#L168)